spec/rubocop/cop/root_cops/up_and_down_or_change_spec.rb

```diff
@@ -0,0 +1,45 @@
+require_relative "../../../spec_helper.rb"
+
+RSpec.describe RuboCop::Cop::RootCops::UpAndDownOrChange do
+  subject(:cop) { described_class.new }
+
+  context "when the migration does not have #change or #up/#down" do
```

Comment:
Can we add some tests around the `active_record_migration?` matcher? maybe something like

```ruby
context "when the node doesn't inherit from ActiveRecord" do
  it "doesn't report an offense" do
    expect_no_offenses(<<~RUBY.strip_indent)
      class CreatePosts
        def some_method; end
      end
    RUBY
  end
end

Reply:
If you don't have the node matcher you'll actually end up w/ a bunch of failures within this repo–see https://buildkite.com/root-insurance/root-ruby-style/builds/136#0855f498-1df1-4583-8fd3-4ddf59dc759d

Reply:
IMO having an explicit test is clearer than relying on us linting our code as a test of this rule. Especially since we can do things like scoping

Reply:
https://github.com/Root-App/root-ruby-style/pull/31/commits/51a009c813d7a93167ccce8e843b20cd37724555